src/main/java/at/fh/joanneum/irfc/persistence/entiy/PictureEntity.java

```diff
@@ -0,0 +1,39 @@
+package at.fh.joanneum.irfc.persistence.entiy;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import javax.persistence.*;
+
+/**
+ * @author gregor.wakonig@edu.fh-joanneum.at
+ **/
+@Entity
+@NoArgsConstructor
+@Getter
+@Setter
+@Table(name = "picture")
+public class PictureEntity {
+
+    @Id
+    @SequenceGenerator(
+            name = "picture_id_seq",
+            sequenceName = "picture_id_seq",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "picture_id_seq"
+    )
+    @Column(name = "picture_id")
+    private Long pictureId;
+
+    @Column
+    private String title;
+
+
+    @ManyToOne
+    @JoinColumn(name = "fk_event_info", nullable = false)
```

Comment:
Not sure but i think this has to be at least nullable cause not every picture belongs to an EventInfo or not?
@PhilipMoser not sure how those relations should work in Hibernate (can you take a quick look over the relations if this is correct) :)

Reply:
the one to many side should be mappedBy and not annotated with a join column.
you can check out the EventLocationEntity and how it is mapped to the EventEntity and
the explanation in this post https://examples.javacodegeeks.com/java-development/enterprise-java/hibernate/hibernate-foreign-key-example/ is helpful in my opinion. :)

Reply:
done